src/Test.FunctionalTests.Sarif/GitInformationTests.cs

```diff
@@ -1,34 +0,0 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using System;
-using FluentAssertions;
-using Xunit;
-
-namespace Microsoft.CodeAnalysis.Sarif
-{
-    public class GitInformationTests
-    {
-        [Fact]
-        public void GetVersionControlDetails_ReturnsExpectedInformation()
```

Comment:
>GetVersionControlDetails_ReturnsExpectedInformation [](start = 20, length = 51)

Removed because `GitHelper` is no longer responsible for creating `VersionControlDetails`.